src/user_config.rs

```diff
@@ -168,6 +168,7 @@ pub struct KeyBindings {
   pub back: Key,
   pub jump_to_album: Key,
   pub jump_to_artist_album: Key,
+  pub jump_to_context: Key,
```

Comment:
To get the keybind working, we need to add it to the `KeyBindingsString` struct.

```rust
pub struct KeyBindingsString {
  ...
  jump_to_artist_album: Option<String>,
  jump_to_context: Option<String>,
  ...
}
```
And also run the `to_keys` macro, within the `load_keybindings` function.

```rust
to_keys!(jump_to_context);
```

Hope this makes sense? Have a look at `jump_to_artist_album` as an example